src/roadmap/v1/lifecycle/rhel.py

```diff
@@ -134,9 +138,51 @@ async def get_relevant_systems(
                 release_date=release_date,
                 retirement_date=retirement_date,
                 count=count,
+                related=False,
             )
         )
 
+    if related:
+        relateds = set()
+        today = date.today()
+        for count_key, count in system_counts.items():
+            minor = count_key.minor if count_key.minor is not None else -1
+            for key, rhel in OS_LIFECYCLE_DATES.items():
+                rhel_minor = rhel.minor if rhel.minor is not None else -1
+                if rhel.major == count_key.major and rhel_minor > minor and rhel.end > today:
+                    relateds.add(key)
+        relateds -= system_keys
+        for key in relateds:
+            os = OS_LIFECYCLE_DATES[key]
+            results.append(
+                System(
+                    name=os.name,
+                    major=os.major,
+                    minor=os.minor,
+                    lifecycle_type=LifecycleType.mainline,
+                    release_date=os.start,
+                    retirement_date=os.end,
+                    count=0,
+                    related=True,
+                )
+            )
+
+    if len(results) == 0:
```

Comment:
Same comments as above.
```suggestion
    if not results:
```

Reply:
fixed on other branch